### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by anthonygauthier** [mstraughan86/Scry#2]
Hi there @mstraughan86,

I came across this repo and it looks very interesting. I'm wondering though, maybe I misunderstood the README, but it feels like you're kind of reinventing "PLEX". 

Please do correct me if I'm wrong, but what you seek to achieve is the following, right?

- Get your favorite television shows' video files on your own (somewhere on the internet, not to name any specific location 😅 
- Have a page that shows every series/movies you have
- Have a page that shows every episodes in a selected serie
- Have a media player that locally streams the video file

As I said, correct me if I'm wrong! Although it seems like you kind of want to "reinvent the wheel", the project does look very interesting and I find it could be fun to work on 😄 ! 

Having a clear understanding of your vision might help myself and others to figure out if contributing to this project is something that is of interest to us 😃 .

Thanks a lot,

**Comment by mstraughan86** [comment]
Hello @delirius325 thanks for checking out my repository!
In essence, yes! What you describe was what this repository is/was aiming for. This project was originally an attempt at making a flexible Netflix clone, and I must admit that progress on this codebase has slowed down considerably since I've implemented a [homeserver docker-plex setup](https://github.com/mstraughan86/mediabox). Infact, Plex accomplishes everything that I originally wanted.

But during development, I expanded scope for this project to go beyond a video server toward a few directions:
- Site generator: If I can spin up a single TV show server based on a few video files and an IMDB link, I can make any video site I want, and I could in theory deploy it anywhere I want, not just for my home LAN. That could be useful.
- Ad Integration: As an aside, I've been working with digital ads and thought that this would be an excellent test sandbox. Video ad integration is a unique problem to solve and I thought it was relevant given this project content.
- Video File Processing: Much of the wishlist and a lot of my own personal brainstorming has been in this area. I wanted to offer as much raw data as I could extract in a usable form, but not necessarily scraped from the web. Something akin to [Frinkiac](https://frinkiac.com/), but I envisioned more.

There was some more bouncing around in my head, but in general that was the last direction of this repository. You are correct in your assumption that the initial and majority of (97%) the requirements are met very easily by Plex.

Check out [mediabox](https://github.com/mstraughan86/mediabox), I am using it right now!
